nav2_behavior_tree/plugins/action/compute_path_to_pose_action.cpp

```diff
@@ -40,20 +40,23 @@ void ComputePathToPoseAction::on_tick()
 BT::NodeStatus ComputePathToPoseAction::on_success()
 {
   setOutput("path", result_.result->path);
+  setOutput("global_planner_result_code", result_.result->error_code);
```

Comment:
If its successful, it shouldn't have an error code - so set to 0?

Reply:
Sure, 

but shouldn't that just be passed though on the action server side? 

`action_server_->on_success(result)`

I guess it never hurts to specify.